src/components/comment.jsx

```diff
@@ -10,7 +10,7 @@ function ListReviewComments ({review_ID}) {
         fetchReviewComment(review_ID).then((data) =>{
            setComment(data)
            setIsLoading(false)
-           
+                      
            
         })
```

Comment:
Put a catch on this. Currently you have an uncaught 404 error if the review has no comments. You need to catch this error and deal with it. Have a look at /reviews/14 and open your console to see what i mean